Wikipedia/Code/ArticleWebMessagingController.swift

```diff
@@ -548,7 +548,7 @@ extension ArticleWebMessagingController {
             return
         }
         
-        var articleAsLivingDocBoxInnerHTML = "\(articleAsLivingDocBoxInnerContainerHTMLStart)<h4 id='significant-changes-header'>RECENT CHANGES</h4><ul id='significant-changes-list'>"
+        var articleAsLivingDocBoxInnerHTML = "\(articleAsLivingDocBoxInnerContainerHTMLStart)<h4 id='significant-changes-header'>SIGNIFICANT EVENTS</h4><ul id='significant-changes-list'>"
```

Comment:
Should we translate this? While AaaLD is only in English for now, we've used translated strings for everything else.

Reply:
Also: This now says "Significant Events", but the title of the modal is "Recent Changes". Should the modal also be updated?

Reply:
Oh good catch! The article content insert doesn't show small events, so we changed it here since it might be hiding more recent (small) changes. The modal still shows small events, so maybe this is fine. @carolynlimadeo what do you think, should the modal nav bar title and modal header view title label change from "Recent Changes" to "Significant Events" for consistency?

Reply:
@mcleinman on your first point, I just pushed a commit that added translations for this method. We can ask about the headers in planning today.

Reply:
@mcleinman confirmed with Carolyn, she wants to leave the modal headers as "Recent Changes" since it includes small edits, and the article content insert does not. Thanks for checking!